installer/scripts/common/setupIstio.sh

```diff
@@ -1,23 +1,39 @@
 #!/bin/bash
 source ./common/utils.sh
 
-kubectl create namespace istio-system
+kubectl get ns istio-system
+ISTIO_AVAILABLE=$?
+if [[ "$ISTIO_AVAILABLE" == 0 ]] && [[ "$ISTIO_INSTALL_OPTION" == "Reuse" ]]; then
+    # An istio-version is already installed
+    print_info "Istio installation is reused but its compatibility is not checked"
```

Comment:
Would it make sense to perform some basic checks here, same as what we do after installation, e.g.:
```bash
 wait_for_crds "adapters.config.istio.io,attributemanifests.config.istio.io,authorizationpolicies.rbac.istio.io,clusterrbacconfigs.rbac.istio.io,destinationrules.networking.istio.io,envoyfilters.networking.istio.io,gateways.networking.istio.io,handlers.config.istio.io,httpapispecbindings.config.istio.io,httpapispecs.config.istio.io,instances.config.istio.io,meshpolicies.authentication.istio.io,policies.authentication.istio.io,quotaspecbindings.config.istio.io,quotaspecs.config.istio.io,rbacconfigs.rbac.istio.io,rules.config.istio.io,serviceentries.networking.istio.io,servicerolebindings.rbac.istio.io,serviceroles.rbac.istio.io,sidecars.networking.istio.io,templates.config.istio.io,virtualservices.networking.istio.io"
```

Please also state in the output that this option is not supported

Reply:
I do not add this check as we do not know, which CRDs are actually used.